Resources/Locale/en-US/accent/italian.ftl

```diff
@@ -34,7 +34,7 @@ accent-italian-words-9 = could
 accent-italian-words-replace-9 = could-a
 
 accent-italian-words-10 = dad
-accent-italian-words-replace-10 = pappa
+accent-italian-words-replace-10 = papà
```

Comment:
If the accent is not desired for gameplay reasons, even just `papa` would be better, I think:

```suggestion
accent-italian-words-replace-10 = papa
```